src/kia_hyundai_api/us_kia.py

```diff
@@ -279,6 +324,10 @@ async def start_climate(
             defrost: bool,
             climate: bool,
             heating: bool,
+            driver_seat: int = 0,
```

Comment:
instead of int let's add an enum for easier use of this library

add to const.py
```
from enum import Enum
class SeatSettings(Enum):
    NONE = 0
    HeatHigh = 1
    HeatMedium = 2
    HeatLow = 3
    CoolHigh = 4
    CoolMedium = 5
    CoolLow = 6
```
then these can be 
driver_seat: SeatSettings | None = None
etc

Reply:
I'll have to make a couple changes in the other repo to handle this.